scripts/ci_scripts/lighthouserc.js

```diff
@@ -0,0 +1,23 @@
+const _ = require("lodash");
+
+const {
+  route_load_tests_config,
+} = require("../../client/browser-tests/route-load-tests-config.js");
+
+module.exports = {
+  ci: {
+    collect: {
+      numberOfRuns: 1,
+      url: _.map(
+        route_load_tests_config,
+        ({ route }) =>
+          `https://dev.ea-ad.ca/${process.env.CIRCLE_BRANCH}/index-eng.html#${route}`
+      ),
+    },
+    upload: {
+      target: "lhci",
+      serverBaseUrl: "https://infobase-lhci.herokuapp.com/",
+      token: "554e44e8-3fcb-46ba-8444-9cb54850bc72", // could also use LHCI_TOKEN variable instead
```

Comment:
`The build token is used by CI to upload new data to your server. It's not really very secret, especially if you'd like to evaluate the performance of external contributor contributions.`

Reply:
We don't run CI for external contributor fork-PRs though, because of our other more-secret CI env vars. Might as well reissue this token and move it to an CircleCI env var. Worst case, someone could use it to DOS our heroku, or even just feed our lighthouse bad data, right? Would be a petty prank, and I don't expect anyone would, but keeping it secret's not that much more work.